Monika After Story/game/script-topics.rpy

```diff
@@ -3587,6 +3587,35 @@ label monika_chloroform:
     m 1tsb "You're so cute when you're nervous."
     return
 
+init 5 python:
+    if os.path.isfile(os.path.expandvars("%APPDATA%") + '\RenPy\JustYuri\persistent'):
+      addEvent(Event(persistent.event_database,eventlabel="just_yuri",category=['club members'],prompt="Just Yuri mod",random=True))
+        
+label just_yuri:
+       m 1esc "[player]..."
+       m 4esc "A thing drew to my attention."
+       m 4esc "Did you..."
+       m 4eka "Did you installed ''Just Yuri'' mod?"
+       m 2rksdla "It seems you did." 
+       m 2rsc "Um..."
+       m 2eka "You don't cheating on me right?"
+       m 2hksdrb "No, of course you wouldn't do that to me."
+       m 1eka "You probably curious about talking with Yuri too."
+       m 1eka "It is OK."
```

Comment:
I would turn "It is" into a contraction to make "It's".

I might also replace "OK" with "alright".

The "K" in "OK" should be lowercase, should you decide to keep the word.